coderd/database/querier_test.go

```diff
@@ -3587,6 +3588,752 @@ func TestOrganizationDeleteTrigger(t *testing.T) {
 	})
 }
 
+type extTmplVersion struct {
```

Comment:
```suggestion
type templateVersionWithPreset struct {
```

or, just return both objects and pass them around.

Reply:
done